src/containers/App.tsx

```diff
@@ -179,41 +177,52 @@ export const App = () => {
                   }}
                 />
               </div>
-              <button
-                className="btn btn-sm btn-accent m-1 w-3/4"
-                onClick={() => {
-                  setServerToken(DEFAULT_TOKEN);
-                  setHost(DEFAULT_HOST);
-                  setPath(DEFAULT_PATH);
-                  setIsWss(DEFAULT_IS_WSS);
-                  setIsHttps(DEFAULT_IS_HTTPS);
-                }}
-              >
-                Restore default
-              </button>
-              <button
-                disabled={!host || !path || !serverToken}
-                className="btn btn-sm btn-accent m-1 w-3/4"
-                onClick={() => {
-                  setJellyfishServers((prev) => {
-                    return {
-                      ...prev,
-                      [host]: {
-                        host,
-                        isWss,
-                        isHttps,
-                        path,
-                        serverToken,
-                        refetchDemand,
-                        active: activeHost === host,
-                      },
-                    };
-                  });
-                  setActiveHost(host);
-                }}
-              >
-                Connect to server
-              </button>
+              <div className="flex flex-row gap-1 justify-between">
+                <button
+                  className="btn btn-error btn-sm p-1 tooltip tooltip-error tooltip-right"
+                  data-tip="Restore default"
+                  onClick={() => {
+                    setServerToken(DEFAULT_TOKEN);
+                    setHost(DEFAULT_HOST);
+                    setPath(DEFAULT_PATH);
+                    setIsWss(DEFAULT_IS_WSS);
+                    setIsHttps(DEFAULT_IS_HTTPS);
+                  }}
+                >
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    className="h-6 w-6"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                  >
+                    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M6 18L18 6M6 6l12 12" />
```

Comment:
For me, this button looks at the moment more like quit than restore default. I'd change the color to `info` (gray) and the icon to return arrow for example `TbArrowBack` in react-icons)